.github/workflows/release_github.yaml

```diff
@@ -0,0 +1,113 @@
+name: Create a github release for the spark-client Python library
+
+env:
+  BRANCH: ${{ github.ref_name }}
+  VERSION: 0.0.0
+
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+jobs:
+  checks:
+    name: Check build condition for spark libs
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version:
+          - '3.9'
+          - '3.10'
+    steps:
+      - id: checkout
+        name: Checkout repo
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ env.BRANCH }}
+          fetch-depth: 0
+      - id: setup-python
+        name: Setup Python
+        uses: actions/setup-python@v2.2.2
+        with:
+          python-version: ${{matrix.python-version}}
+          architecture: x64
+      - id: code-checks
+        name: Make sure the code follows conventions and tests correctly
+        run: |
+          make checks
```

Comment:
These "checks" seeems a bit of a copy and paste from the`ci.yaml`. We could re-use workflows, e.g. [here](https://github.com/canonical/kafka-operator/blob/main/.github/workflows/release.yaml#L29)

Reply:
Beside the normal CI checks, I'm thinking that we should definitely do a check where we make sure that the tag version and the VERSION we are release (information in pyproject.toml) match. To prevent to have inconsistency between tag and version, and also making sure the same version cannot be released more than once. 

This could of course be a separate ticket or tech dept to be placed in the backlog, for further improvement

Reply:
Sure, I'll move it here.

Reply:
I've split `ci.yaml` to two workflows (one referring the other), seemed best.